guides/development-workflow.md

```diff
@@ -461,26 +445,32 @@ story and the acceptance criteria that should be met.
 
 A good pull request should:
 
-- be created by the authoring developer
-- meet all of the acceptance criteria on the associated story
-- focus on the single problem at hand, not doing so will usually increase the
-  time taken to get it merged
-- include appropriate detail to assist the reviewer, this includes links to
-  relevant content and a duplication of anything of note in the commit
-  descriptions
-- update the [changelog](#tracking-changes) if it's a significant change
-
-```
-# Poor
-A new registration form and fix a bug with contacts
-
-# Great
-[123456789] New users can now register for an account
-
-- This has been rate limited, would like your feedback on the values I've chosen
-- Passwords are encrypted using BCrypt, so I've taken this opportunity to
-  increase the default rounds from 8 to 12
-```
+- describe the problem without assuming the reader has existing context
+- highlight the changes being made
+- describe and bring attention to particular difficulties or areas that you
+  particularly want the opinion of the reviewer
+- for user facing changes include screenshots of the before and after
+- the changes should be focused on meeting the acceptance criteria
```

Comment:
This is less prescriptive than the previous version, which is good. I think it could be more encouraging of PRs which address only part of the acceptance criteria for a card. Maybe it's about being clear which acceptance criteria the changes meet?

For example, a refactoring which will enable the feature work to be done, but doesn't actually meet any acceptance criteria yet. It's good to ship that as its own thing (as per the point below) rather than waiting until the feature is built on top of it.

Or, after realising part way through development that the feature could be shipped as a two part change, being clear that the first PR only meets one or two of the acceptance criteria.

Reply:
Good points, will add more 👍